src/Sylius/Bundle/ShopBundle/Resources/config/routing/checkout.yml

```diff
@@ -85,7 +85,10 @@ sylius_shop_checkout_complete:
             state_machine:
                 graph: sylius_order_checkout
                 transition: complete
-            redirect: sylius_shop_thank_you
+            redirect:
+                route: sylius_shop_order_pay
+                parameters:
+                    lastNewPaymentId: expr:service('sylius.context.cart').getCart().getLastNewPayment().getId()
```

Comment:
We should probably provide order id here, since later we will save order id in the session for the thank you page.